drivetrain.py

```diff
@@ -43,7 +43,10 @@ def set_neutral(self):
         self.set_servo_pulse(self.channels['right'], self.servo_mid)
 
     # TODO - flesh out setters for raw pulse values (both channels)
-    def mix_channels(self, pulse_throttle, pulse_steering):
+    def mix_channels(self, throttle, steering):
```

Comment:
Agree that comment is required, perhaps `mix_channels` is also not a descriptive enough method name now. Suggest `mix_channels_and_assign`?